.github/workflows/gosec.yaml

```diff
@@ -0,0 +1,25 @@
+name: Gosec
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: on
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version: '1.22'
+          cache: false
+      - name: Run Gosec
+        uses: securego/gosec@master
```

Comment:
This is always running at the latest version of GoSec yeah ?

Reply:
yes, that's also the suggest way to structure the job taken from the [gosec page](https://github.com/marketplace/actions/gosec-security-checker)